source/renderer/app/themes/daedalus/light-blue.js

```diff
@@ -491,6 +491,11 @@ export default {
   '--theme-progress-bar-background-color': 'rgba(255, 255, 255, 0.3)',
   '--theme-progress-bar-foreground-color': 'rgba(255, 255, 255, 0.7)',
 
+  '--theme-block-decentralization-progress-stripe-dark-1-background-color':
```

Comment:
please, remove the word `block` from this var. It would be good to start with `--theme-staking-decentralization-progress...` so we can add the same prefix for the other pages.